app/helpers/application_helper.rb

```diff
@@ -118,7 +118,7 @@ def team_group_emails(team)
 
   def links_to_user_teams(user)
     user.teams.map do |team|
-      link_to(team.name || team.project, team, class: "team #{team.sponsored? ? 'sponsored' : ''}")
+      link_to(team.name || team.project, team, class: "team #{team.accepted? ? 'sponsored' : ''}")
```

Comment:
Hm, did you not just rename the `sponsored` css class to `full-time`... 🤔 
I fear this then won't work anymore.